client/js/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -86,31 +86,42 @@ export default {
         cardRank = this.selectedCard.rank;
       }
       // Re-usable move-objects
+      const allMovesAreDisabled = !this.isPlayersTurn || this.frozenId === this.selectedCard.id;
+      const allMovesDisabledExplanation = !this.isPlayersTurn
+        ? "It's not your turn"
+        : 'This card is frozen';
+
+      let disabledText;
+      if (allMovesAreDisabled) {
+        disabledText = allMovesDisabledExplanation;
+      }
+
       // Points
       const pointsDescription = `Gain ${cardRank} point` + (cardRank === 1 ? '' : 's');
       const pointsMove = {
         displayName: 'Points',
         eventName: 'points',
         moveDescription: pointsDescription,
-        disabled: !this.isPlayersTurn,
-        disabledExplanation: "It's not your turn",
+        disabled: allMovesAreDisabled,
+        disabledExplanation: disabledText,
       };
       // Scuttling
       const scuttleDisabled = !this.isPlayersTurn || !this.hasValidScuttleTarget;
       let scuttleDisabledExplanation = 'You can only scuttle smaller point cards';
       if (this.$store.getters.opponent.points.length === 0) {
         scuttleDisabledExplanation = 'Your opponent has no point cards to scuttle';
       }
-      if (!this.isPlayersTurn) {
-        scuttleDisabledExplanation = "It's not your turn";
+      if (allMovesAreDisabled) {
+        scuttleDisabledExplanation = allMovesDisabledExplanation;
       }
```

Comment:
I feel like we could clean this up a bit too to prioritize the "It's not your turn"/"Card is frozen" text for scuttle moves. Im thinking:
`if (allMovesAreDisabled) {
        scuttleDisabledExplanation = allMovesDisabledExplanation;
      } else if (this.$store.getters.opponent.points.length === 0) {
        scuttleDisabledExplanation = 'Your opponent has no point cards to scuttle';
      }`

Reply:
I'm open to it. I'd like to open a new issue to refactor this function so every move is it's own computed property and the `moves()` computed prop just combines them all. I think that'd be cleaner. Thoughts @TommySchmid and @seriouslysean? If so then we could handle that change in the file-wide rework

Reply:
Works for me!